lib/domgen/ejb/templates/aggregate_service_test.java.erb

```diff
@@ -1,16 +1,15 @@
 /* DO NOT EDIT: File is auto-generated */
-package <%= to_package(repository.ejb.qualified_aggregate_service_test_name) %>;
+package <%= data_module.ejb.server_service_package %>;
```

Comment:
I would use the same pattern as used everywhere else and use `to_package`. This allows us to override the names of classes and packages of files. Very occasionally we need to do this for one reason or another which is why we use `to_pacage` everywhere else